media/css/m24/components/footer-refresh.scss

```diff
@@ -254,85 +248,44 @@ $max-footer-content-width: $content-max;
 .moz24-footer-donate {
     position: relative;
     border-radius: 0;
-    font-weight: 500;
+    font-weight: 600;
     background-color: $m24-color-white;
     color: $m24-color-black;
     padding: 6px 24px;
     border: $border-width solid $m24-color-black;
     text-decoration: none;
     text-align: center;
     max-width: 800px;
+    margin-bottom: $spacer-lg;
 
     &:link,
     &:visited {
         text-decoration: none;
     }
 
     &:hover,
-    &:visited:hover {
+    &:visited:hover,
+    &:focus,
+    &:focus-visible,
+    &:visited:focus-visible {
+        color: $m24-color-white;
```

Comment:
**Issue:** (blocking) The donate button text and icon stay white on hover in Safari.

Reply:
This is a mystery to me but it explains the span for the text... it must be clashing with some link/button styles defined elsewhere. We'll have to make proper global button styles for the rebrand later but for now I've used the spans for the specificity on this one button.

Reply:
✅